angularjs/tagmanagerTrackingCode/tagmanager.controller.js

```diff
@@ -187,7 +189,10 @@
             if (val !== oldVal) {
                 self.onSiteChange();
             }
-        });
 
+            if (val === oldVal && self.firstTime) {
+                self.onSiteChange();
+            }
+        });
```

Comment:
Do you think it would be better to combine the two `if`s into `if (val !== oldVal || self.firstTime)`

Reply:
you're right, that would be the same, but simpler. I just added this new check separately. I'll combine those.